apis/locale/scripts/generate-all.mjs

```diff
@@ -47,12 +48,6 @@ for (const file of LOCALES_FILES) {
     }
     acc[key].locale[locale] = content[curr].value;
     const localeObj = acc[key].locale[locale];
-    Object.keys(acc[key].locale[locale])
-      .sort()
-      .reduce((a, c) => {
-        a[c] = localeObj[c]; // eslint-disable-line no-param-reassign
-        return a;
-      }, localeObj);
```

Comment:
If someone can figure out the point of these lines I would be happy.
From what I can tell it takes a word, gets its "keys", so 0,1,2,3,4,5 etc, sorts those (?) then runs a reducer over the word. It then assigns in (or tries to, new node doesn't like it) the characters from that same word into it.
There is no difference in the output files between master and this branch, so I have no idea what the point is.